_config.yml

```diff
@@ -262,3 +262,16 @@ collections:
     published: 2019-11-05
     author: Ming Fu <xiao.fu@utas.edu.au>
 
+  cloudstor:
+    output: true
+    permalink: /:collection/:name
+    title: AARNET Cloudstor
+    summary: "Learn to use AARNET Cloudstor in Nectar's Virtual Machine"
+    category: Beginner
+    tags:
+      - tutorial
+    difficulty: 1 # number from 1 to 5
+    duration: 21 # number of minutes
+    status: draft # draft or published
+    published: 2019-11-13
+    author: Ming Fu <xiao.fu@utas.edu.au>
```

Comment:
Notes about your screenshots:
 - Try to avoid using your own username and password. For the purposes of the screenshot, use some dummy name like ```Your Username <your.username@example.com>``` for example
 - Your screenshot windows are far too wide. When they get published, it's very hard to actually make out what is in them on the website. Please make them the minimum required size to see the contents